src/LRS.js

```diff
@@ -893,34 +917,25 @@ TinCan client library
 
         @method _assignAttachmentContent
         @private
-        @param {Array} [stmts] Array of TinCan Statement JSON objects
-        @param {Array} [headers] Array containing section headers from the HTTP response
-        @param {String} [content] The content to place into its attachment
-        @return {Array} Array of TinCan Statement JSON objects with correctly assigned attachment content
+        @param {Array} [stmts] Array of TinCan.Statement JSON objects
+        @param {Hashmap} [content] Hashmap of the content to place into its attachment
```

Comment:
Is "Hashmap" a valid parameter type in these docs? I think we can stick with object in the JS world.